apps/src/code-studio/components/progress/ProgressDot.story.jsx

```diff
@@ -211,36 +211,16 @@ export default storybook => {
         )
       },
       {
-        name: 'named level in course overview',
-        story: () => (
-          <ProgressDot
-            courseOverviewPage={true}
-            saveAnswersBeforeNavigation={false}
-            status="not_tried"
-            level={{
-              icon: 'fa-file-text',
-              ids: [5442],
-              kind: LevelKind.named_level,
-              name: 'CSP Pre-survey',
-              position: 1,
-              previous: [1,1],
-              title: 1,
-              url: 'http://localhost-studio.code.org:3000/s/csp1/stage/1/puzzle/1'
-            }}
-          />
-        )
-      },
-      {
```

Comment:
Removed because this became identical to the previous item (kind: LevelKind.puzzle and has a name).